modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -82,6 +88,9 @@ public class PartitionReplicaListener implements ReplicaListener {
     /** Dummy primary index. */
     private final ConcurrentHashMap<ByteBuffer, RowId> primaryIndex;
 
+    /** Cursors map. */
+    private final ConcurrentNavigableMap<IgniteUuid, Cursor<BinaryRow>> cursors;
```

Comment:
Please add detailed javadoc, denoting that IgniteUuid is (UUID)txId + (long)cursorId

Reply:
Done.